code/modules/jobs/job_types/geneticist.dm

```diff
@@ -9,6 +9,8 @@
 	supervisors = "the chief medical officer and research director"
 	selection_color = "#d4ebf2"
 	chat_color = "#83BBBF"
+	minimal_player_age = 7
+	exp_requirements = 600
```

Comment:
Im sorry what.
10 hours to play geneticist.
This is one of the first roles I played and an amazing role for new players considering the little outside interaction so you can learn at your own pace.

Reply:
This shall be moved down to the prior group, but I'm still against open access *mainly* due to hulk.